.github/workflows/changelog.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Changelog
+on:
+  release:
+    types: [published]
+jobs:
+  updateChangelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Update Changelog
+        run: |
+          curl -o git-chglog.tar.gz -L https://github.com/git-chglog/git-chglog/releases/download/v0.15.1/git-chglog_0.15.1_darwin_amd64.tar.gz
+          tar -zxvf git-chglog
```

Comment:
Is this `tar` command correct? should it be `tar -xvf git-chglog.tar.gz`?

Reply:
Yes the command is correct -z it is for gzip'ed tar files.